### ISSUE TITLE
Fix for Tensorflow 2.xx load_model bug

### ISSUE THREAD

**Issue by wjguan** [PolarBean/DeepSlice#51]
The model is unable to load_weights in tensorflow 2.xx versions because of the removal of the _layers attribute of keras Models. 

The `base_model._layers.pop()` call in the original code only cosmetically removes the last two layers in the Xception model, but during the actual execution of the graph, these last two layers (the AveragePooling and Softmax layers) are still being performed. Thus, the output shape of the Xception layer in the overall DSModel.model is still (1000,) and correctly interfaces with the rest of the model. However, if you try to access `base_model.layers` or even `base_model.summary()`, these last two layers are hidden and nowhere to be seen.

My workaround is, instead of using `model.load_weights()`, we must manually set each of the layers using `model.layers[idx].set_weights(list_of_numpy_weights)` after loading the weights in with `h5py`. I wrote this function to be included in the `neural_network.py` module. It should be called whenever `model.load_weights()` was being called:

```
def load_xception_weights(model, weights):
    with h5py.File(weights, "r") as new:
        # set weight of each layer manually
        model.layers[1].set_weights([new["dense"]["dense"]["kernel:0"], new["dense"]["dense"]["bias:0"]])
        model.layers[2].set_weights([new["dense_1"]["dense_1"]["kernel:0"], new["dense_1"]["dense_1"]["bias:0"]])
        model.layers[3].set_weights([new["dense_2"]["dense_2"]["kernel:0"], new["dense_2"]["dense_2"]["bias:0"]])

        # Set the weights of the xception model 
        weight_names = new["xception"].attrs["weight_names"].tolist()
        weight_names_layers = [name.decode("utf-8").split("/")[0] for name in weight_names]

        for i in range(len(model.layers[0].layers)):
            name_of_layer = model.layers[0].layers[i].name
            # if layer name is in the weight names, then we will set weights
            if name_of_layer in weight_names_layers:
                # Get name of weights in the layer
                layer_weight_names = []
                for weight in model.layers[0].layers[i].weights:
                    layer_weight_names.append(weight.name.split("/")[1])
                h5_group = new["xception"][name_of_layer]
                weights_list = [np.array(h5_group[kk]) for kk in layer_weight_names]
                model.layers[0].layers[i].set_weights(weights_list)
    return model
```

I can make a pull request with the full fix if you'd like.

**Comment by wjguan** [comment]
The pull request to address this issue is PR #52
